### PR TITLE
Translation Delivery

### PR DESCRIPTION
**Pull request by service-tip-git**
Translation Delivery

### CODE REVIEW

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them,  @I534472

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -58,15 +58,15 @@
...
+"ExposureDetection_High" = "Increased Risk";
 
 "ExposureDetection_NumberOfDaysStored" = "%d of 14 days active";
 
 "ExposureDetection_Refreshed" = "Updated: %@";
 
 "ExposureDetection_Refreshed_Never" = "Unknown";
 
-"ExposureDetection_RefreshingIn" = "Update in %02d:%02d\U00A0minutes";
+"ExposureDetection_RefreshingIn" = "Update in %02d:\U00A0minutes";
```
Comment (MichaelPFrey) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:69]: Can someone explain that change?

even assuming you want to remove displaying unnecessary information: why remove the `%02d` but then keeping the `:` ?

Note that this change is also in the other translations of this PR.

For german see:

https://github.com/corona-warn-app/cwa-app-ios/blob/develop/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L69

https://github.com/corona-warn-app/cwa-app-ios/blob/95e7ea31ba48b92db3f72d55eefbccbb3a5bb98e/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L69

File: src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings [hunk 2]
```diff
@@ -66,7 +66,7 @@
 
 "ExposureDetection_Refreshed_Never" = "Неизвестен";
 
-"ExposureDetection_RefreshingIn" = "Актуализиране след %02d:%02d\U00A0минути";
+"ExposureDetection_RefreshingIn" = "Актуализиране след %02d:\U00A0минути";
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings:69]: Is this change intended?

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 1 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:69]: good catch...

File: src/xcode/ENA/ENA/Resources/Localization/pl.lproj/Localizable.strings [hunk 3]
```diff
@@ -66,7 +66,7 @@
 
 "ExposureDetection_Refreshed_Never" = "Nieznane";
 
-"ExposureDetection_RefreshingIn" = "Aktualizuj za %02d:%02d\U00A0minut(y)";
+"ExposureDetection_RefreshingIn" = "Aktualizacja za %02d:\U00A0minut(y)";
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/pl.lproj/Localizable.strings:69]: Again this change? might be a mistake?

File: src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.strings [hunk 4]
```diff
@@ -66,7 +66,7 @@
 
 "ExposureDetection_Refreshed_Never" = "Bilinmiyor";
 
-"ExposureDetection_RefreshingIn" = "%02d:%02d\U00A0minutes içinde güncelle";
+"ExposureDetection_RefreshingIn" = "%02d:\U00A0minutes içinde güncelle";
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.strings:69]: ... here

**Review by steins-code — CHANGES_REQUESTED** [review]
Check changes regarding "Aktualisierung in %02d:%02d\U00A0Minuten"; in other languages

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them,  @I534472

**Review by steins-code — APPROVED** [review]

**Review by haosap — APPROVED** [review]
